src/App.tsx

```
@@ -1,24 +1,179 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import "antd/dist/antd.min.css";
+import "./App.css";
+import { Recipe } from "./recipe";
+import { Button, Collapse, Image } from "antd";
+import { DeleteOutlined, EditOutlined } from "@ant-design/icons";
 
 function App() {
+  const baseUrl = `${process.env.REACT_APP_API_URL}`;
```

This actually swallows the error because if the env var is undefined, you the string "undefined"